geode-redis/README.md

```diff
@@ -128,210 +128,55 @@ not connected>
-|                    	|                                                    	| ZINTERSTORE                   	|
-|                    	|                                                    	| ZLEXCOUNT                     	|
-|                    	|                                                    	| ZPOPMAX                       	|
-|                    	|                                                    	| ZPOPMIN                       	|
-|                    	|                                                    	| ZRANGE                        	|
-|                    	|                                                    	| ZRANGEBYLEX                   	|
-|                    	|                                                    	| ZRANGEBYSCORE                 	|
-|                    	|                                                    	| ZRANK                         	|
-|                    	|                                                    	| ZREM                          	|
-|                    	|                                                    	| ZREMRANGEBYLEX                	|
-|                    	|                                                    	| ZREMRANGEBYRANK               	|
-|                    	|                                                    	| ZREMRANGEBYSCORE              	|
-|                    	|                                                    	| ZREVRANGE                     	|
-|                    	|                                                    	| ZREVRANGEBYSCORE              	|
-|                    	|                                                    	| ZREVRANK                      	|
-|                    	|                                                    	| ZSCAN                         	|
-|                    	|                                                    	| ZSCORE                        	|
-|                    	|                                                    	| ZUNIONSTORE                   	|                   	|
+The Geode APIs compatible with currently implements a subset of the full Redis command set.
+
+#### <a name="redis-command-status"></a> Supported Commands Compatable With Redis [Return to top](#introduction)
+- EXPIRE
+- EXPIREAT
+- GET
+- GETRANGE
+- HDEL
+- HEXISTS
+- HGET
+- HGETALL
+- HINCRBY
+- HINCRBYFLOAT
+- HLEN
+- HMGET
+- HMSET
+- HSCAN
+- HSET
+- HSETNX
+- HSTRLEN
+- HVALS
+- HKEYS
+- INCR
+- INCRBY
+- INCRBYFLOAT
+- INFO
+- KEYS
+- MGET
+- PERSIST
+- PEXPIRE
+- PEXPIREAT
+- PING
+- PSUBSCRIBE
+- PTTL
+- PUBLISH
+- PUNSUBSCRIBE
+- QUIT
+- RENAME
+- SADD
+- SET
+- SETNX
+- SLOWLOG <sup>1</sup>
+- SMEMBERS
+- SREM
+- STRLEN
+- SUBSCRIBE
+- TTL
+- TYPE
+- UNSUBSCRIBE
 
 **NOTES:**
-
-[1] - UNLINK is implemented as a synonym to DEL and does not unlink asynchronously.
-[2] - SLOWLOG is implemented as a NoOp.
+<sup>1</sup>  SLOWLOG is implemented as a NoOp.
```

Comment:
I like the superscript! You could put `<br>` after `NOTES` so that the note is on the next line (rather than on the same line as `NOTES`).

Reply:
sure.  added.